agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -2562,17 +2365,6 @@ bool slave_thread::handle_cmdu_monitor_message(
             iface_status_operational_state = false;
         }
 
-        if (slave_state == STATE_OPERATIONAL && notification_in->new_tx_state() == 0 &&
-            notification_in->new_hostap_enabled_state() == 1) {
-            if (!set_wifi_iface_state(config.hostap_iface, WIFI_IFACE_OPER_ENABLE)) {
-                LOG(ERROR) << "error enabling hostap tx --> slave_reset();";
-                platform_notify_error(BPL_ERR_SLAVE_IFACE_CHANGE_STATE_FAILED,
-                                      config.hostap_iface.c_str());
```

Comment:
Actually, bpl_cfg_notify_error() is not implemented in any bpl implementation, so maybe all that stuff can be removed as well.

Reply:
It was implemented on UGW bpl if I remember correctly. Who removed it?
@tomereli what to do with it? If we remove it, I will need to drop the commit that touched the `bpl_err.h` and remove this file.

Reply:
different PR